### PR TITLE
HackCU V updates

### PR DESCRIPTION
**Pull request by peasant98**
## What this PR does / why we need it

Just updating the style of the live page to fit the HackCU V page. 

## Related Issues

Fixes #

## Some questions

- [X] I have read the contributing guidelines
- [X] I abide by this repository Code of Conduct

## Special notes for your reviewer

(optional)

## Additional Notes

Do you want to add anything else? We :heart: to hear your opinions!

I tried

### CODE REVIEW

File: _config.yml [hunk 1]
```diff
@@ -1,22 +1,25 @@
...
+sass:
+    sass_dir: assets/_sass
+    style: compressed
+
+timezone: America/Denver
+
+# Countdown Timer
+# Time format: "MONTH DAY YEAR HH:MM:SS TIMEZONE"
+timer_enable: true
+start_time: "February 23 2019 08:00:00 GMT-0700"
```
Comment (casassg) [inline _config.yml]: Hacking starts at 12
```suggestion
start_time: "February 23 2019 12:00:00 GMT-0700"
```

File: _config.yml [hunk 2]
```diff
@@ -1,22 +1,25 @@
...
+end_time: "February 24 2019 15:00:00 GMT-0700"
```
Comment (casassg) [inline _config.yml]: And ends at 12 next day
```suggestion
end_time: "February 24 2019 12:00:00 GMT-0700"
```

File: _sections/events.html [hunk 3]
```diff
@@ -1,40 +1,40 @@
...
-				{% endif %}
-			</li>
-		{% endfor %}
-		</ul>
-	</div>
-</div>
+---
+layout: base
+title: Events
+description: Intro workshops and side events at Hack CU V
```
Comment (casassg) [inline _sections/events.html]:
```suggestion
description: Intro workshops and side events at HackCU V
```

File: _sections/hardware.html [hunk 4]
```diff
@@ -1,57 +1,57 @@
...
-        })
-    });
-}
-refresh();
-setInterval(refresh, 60000);
-
+---
+layout: base
+title: Hardware
+description: Live hardware for Local Hack Day
```
Comment (casassg) [inline _sections/hardware.html]:
```suggestion
description: Live hardware for HackCU V
```

File: _config.yml [hunk 5]
```diff
@@ -1,22 +1,25 @@
...
-
-timezone: America/Denver
-
-# Countdown Timer
-# Time format: "MONTH DAY YEAR HH:MM:SS TIMEZONE"
-timer_enable: true
-start_time: "December 1 2018 09:00:00 GMT-0700"
-end_time: "December 1 2018 21:00:00 GMT-0700"
+original_repo: https://github.com/hackcu/live
+title: Hack CU V Live
```
Comment (casassg) [inline _config.yml]:
```suggestion
title: HackCU V Live
```

File: index.html [hunk 6]
```diff
@@ -1,61 +1,60 @@
...
+</div>
+<div class="row">
+	<div class="col s12 l8">
+		<div class="card accent-color white-text">
+			<div class="card-content">
+				<span class="card-title">
+					General Info
+				</span>
+                    <p><b><i aria-hidden="true" class="fa fa-chevron-right"></i> Slack: <a href="https://hackcu.slack.com/" target="_blank">hackcu.slack.com</a></b><br>Join the HackCU Slack to talk with all the hackers attending our event!</p><br>
+                    <p><b><i aria-hidden="true" class="fa fa-upload"></i> Devpost: <a href="{{site.devpost}}" target="_blank">{{site.devpost}}</a></b><br>Register for Hack CU on Devpost, and submit your hack here by 12:00 pm <b>(submit early, you can edit your submission!)</b></p><br>
```
Comment (casassg) [inline index.html]:
```suggestion
                    <p><b><i aria-hidden="true" class="fa fa-upload"></i> Devpost: <a href="{{site.devpost}}" target="_blank">{{site.devpost}}</a></b><br>Register for HackCU on Devpost, and submit your hack here by 12:00 pm <b>(submit early, you can edit your submission!)</b></p><br>
```

**Review by casassg — CHANGES_REQUESTED** [review]
Hack CU -> HackCU

added a few changes on text and more.

**Comment by peasant98** [comment]
updates lgtm.

**Review by casassg — APPROVED** [review]

**Pull request merged**